### PR TITLE
modules: add basic multiport support

### PR DESCRIPTION
**Pull request by Adphi**

### CODE REVIEW

File: match_modules.go [hunk 1]
```diff
@@ -385,6 +387,84 @@ func (p *Parser) parseTcp(f *map[string]Flag) (state, error) {
...
+						Values: p.parsePorts(),
+					}
+					s = sStart
+				default:
+					// The end of the match statement is reached.
+					// Since we already scanned the ! charackter,
+					// we have to return the sNot state, or
+					// unscanIgnoreWhitespace twice (this can fail
+					// because of a fixed sized buffer, that is full
+					// of Whitespaces).
```
Comment (leonnicolas) [inline match_modules.go]:
```suggestion
```

I think we can remove the comment as we already return the `sNot` State. The comment is wrong at the moment. (probably at other places too)

File: match_modules.go [hunk 2]
```diff
@@ -385,6 +387,84 @@ func (p *Parser) parseTcp(f *map[string]Flag) (state, error) {
...
+					p.unscan(1) // IgnoreWhitespace(2) // unscan 2
```
Comment (leonnicolas) [inline match_modules.go]:
```suggestion
					p.unscan(1)
```

**Review by leonnicolas — COMMENTED** [review]
With basic, you mean we don't support ranges of ports like `1000:2000`?

Yes, this seems hard to represent with the current model. I think if it is helpful without that feature, we can still go ahead and merge.

**Comment by Adphi** [comment]
Yes, that was the idea. I don't have an immediate need for range support.
But as it says in the man page, the range ports should be considered as two ports only... I'm not sure how this would be implemented... 
```
multiport

This module matches a set of source or destination ports. Up to 15 ports can be specified. A port range (port:port) counts as two ports. It can only be used in conjunction with -p tcp or -p udp.
--source-ports [!] port[,port[,port:port...]]
Match if the source port is one of the given ports. The flag --sports is a convenient alias for this option.
--destination-ports [!] port[,port[,port:port...]]
Match if the destination port is one of the given ports. The flag --dports is a convenient alias for this option.
--ports [!] port[,port[,port:port...]]
Match if either the source or destination ports are equal to one of the given ports.
```

**Review by leonnicolas — APPROVED** [review]

**Comment by Adphi** [comment]
Thanks ! 😁

**Pull request merged**